docs/copied-from-beats/command-reference.asciidoc

```diff
@@ -0,0 +1,582 @@
+//////////////////////////////////////////////////////////////////////////
+//// This content is shared by all Elastic Beats. Make sure you keep the
+//// descriptions here generic enough to work for all Beats that include
+//// this file. When using cross references, make sure that the cross
+//// references resolve correctly for any files that include this one.
+//// Use the appropriate variables defined in the index.asciidoc file to
+//// resolve Beat names: beatname_uc and beatname_lc
+//// Use the following include to pull this content into a doc file:
+//// include::../../libbeat/docs/command-reference.asciidoc[]
+//////////////////////////////////////////////////////////////////////////
+
```

Comment:
I don't think that we should mention `machine learning` here.

Reply:
fixed here: https://github.com/elastic/beats/pull/6184/commits/91b8c7be5e6033236f21a0b4c39cbdce2d559111